app/lib/Config.scala

```diff
@@ -37,3 +42,32 @@ object Config {
     }
   }
 }
+
+
+object S3UploadAppConfig extends Config {
+  val bucketName = properties.getOrElse("s3.bucket", "s3-uploader-dev-bucket")
+  val region = properties.getOrElse("aws.region", "eu-west-1")
+  val s3Client = AmazonS3ClientBuilder.standard().withRegion(region).withCredentials(awsCredentials).build()
+  val prettyUrl = "https://uploads.guim.co.uk"
+
+  def key(fileName: String) = {
+    val now = Calendar.getInstance().getTime
+    val dateFormat = new SimpleDateFormat("yyyy/MM/dd")
+    val getCurrentDate = dateFormat.format(now)
+    s"$getCurrentDate/${fileName.replace(' ', '_')}"
+  }
+}
+
+
+object ChartsToolConfig extends Config {
+  val bucketName = properties.getOrElse("s3.chartBucket", "gdn-cdn")
```

Comment:
Should we create a dev bucket, to use instead of `gdn-cdn`?

Reply:
Yeah. That's probably the best thing to do - with an If statement depending on Stage?

Reply:
Actually, because "embed" is disabled in non-Prod environments in the Charts Tool, we shouldn't have any files being stored from the CODE / DEV environments. Leaving for now